addons/mail/static/src/components/message_context_menu/message_context_menu.scss

```diff
@@ -0,0 +1,9 @@
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_MessageContextContainer {
+    height: 40% !important;
+    width: 25% !important;
+    
+}
```

Comment:
Haven't checked in details actual visual of it.
Certainly the styling should just be something like `.o_Dialog_componentMediumSize`

Reply:
As you said in this [comment](https://github.com/odoo/odoo/pull/104557#discussion_r1009330685), It is `o_MessageContextMenu_container` now.

Reply:
Indeed name becomes `o_MessageContextMenu_container` from other comment.
I haven't checked the visual for this context menu right now.
Just a comment to tell that maybe should not define explicit styling here but instead reuse another existing rule, such as this `o_Dialog_componentMediumSize`.

You can ignore this comment for the moment. When the rest of diff is good, we'll see what we do with this part :)